src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Buttons } from '../../constants/common.constants';
+import classes from './SearchBar.module.scss';
+
+interface ISearchBarProps {
+  text: string;
+  onTextChange: (value: string) => void;
+}
+
+class SearchBar extends React.Component<ISearchBarProps, object> {
```

Comment:
do u need the second argument in generic type?

Reply:
no, removed)